scss/tech.scss

```diff
@@ -0,0 +1,66 @@
+$tech-box-border-radius: 10px;
+$tech-box-height: 450px;
+$tech-box-width: 300px;
+$tech-logo-size: 100px;
+$tech-text-margin: 20px;
+
+.tech-box {
+  display: flex;
+  flex-direction: column;
+  float: left;
+  height: $tech-box-height + 20;
+  margin-left: 10px;
+  padding: 40px;
+  position: relative;
+  text-align: center;
+  width: $tech-box-width + 20;
+
+  &::before {
+    content: '';
+    background-color: $white;
+    border: 1px solid $green;
+    border-radius: $tech-box-border-radius;    
+    height: $tech-box-height;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: $tech-box-width;
+    z-index: -2;
+  }
+
+  &::after {
```

Comment:
I think you can move these style definitions to the `tech-box` itself. What do you think? 🙂